app/models/project.rb

```diff
@@ -99,6 +99,9 @@ class Project < ApplicationRecord
      Descriptor
      ProjectMember
      Download
+     DatasetRecordField
```

Comment:
These must be before OriginRelatinship I think, and any other dependants.

Reply:
Shouldn't relationships be destroyed first? Imported objects IDs are stored in json metadata so CollectionObject, CollectingEvent, etc. can be destroyed even if the import dataset is still referencing them somewhere.

Reply:
You are right.